app/src/main/java/com/duckduckgo/app/browser/omnibar/animations/LottiePrivacyShieldAnimationHelper.kt

```diff
@@ -54,10 +54,10 @@ class LottiePrivacyShieldAnimationHelper @Inject constructor(val appTheme: AppTh
             UNKNOWN -> {
                 Timber.i("Shield: UNKNOWN")
             }
-            WARNING -> {
-                val res = if (appTheme.isLightModeEnabled()) R.raw.unprotected_shield else R.raw.dark_unprotected_shield
+            MALICIOUS -> {
+                val res = R.raw.alert_red
```

Comment:
Asked in https://app.asana.com/0/1203867924525323/1209269732493260 about the single file for both themes.

Reply:
Good catch, I initially thought we probably didn't need to do anything, but I had forgotten about the different shade of red. Thanks!